File: backend-go/internal/modules/system/entity/sys_dict_type.go
package entity

import (
	"drip-admin/backend-go/internal/common"
	"time"
)

type SysDictType struct {
	ID        common.Int64String `gorm:"column:id;primaryKey" json:"id"`
	DictName  string             `gorm:"column:dict_name" json:"dictName"`
	DictCode  string             `gorm:"column:dict_code" json:"dictCode"`
	Status    int                `gorm:"column:status" json:"status"`
	Builtin   int                `gorm:"column:builtin" json:"builtin"`
	Remark    *string            `gorm:"column:remark" json:"remark"`
	CreatedAt *time.Time         `gorm:"column:created_at" json:"createdAt"`
	UpdatedAt *time.Time         `gorm:"column:updated_at" json:"updatedAt"`
}

func (SysDictType) TableName() string { return "sys_dict_type" }

File: backend-python/app/modules/system/entity/sys_dict_type.py
from sqlalchemy import BigInteger,Integer,String
from sqlalchemy.orm import Mapped,mapped_column
from app.modules.system.entity.base import Base,TimestampMixin
class SysDictType(TimestampMixin,Base):
 __tablename__="sys_dict_type"
 id:Mapped[int]=mapped_column(BigInteger,primary_key=True)
 dict_name:Mapped[str]=mapped_column(String(64))
 dict_code:Mapped[str]=mapped_column(String(64))
 status:Mapped[int]=mapped_column(Integer)
 builtin:Mapped[int]=mapped_column(Integer)
 remark:Mapped[str|None]=mapped_column(String(255),nullable=True)
